src/todo-list/components/TodoItem.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect} from 'react';
+import styled from 'styled-components';
+import useTodoList from '../hooks/useTodoList';
+
+function TodoItem ({...props}) {
+
+    const { deleteTodo, changeTodoStatus } = useTodoList();
+    const id = props.todo.id;
+    const todoContent = props.todo.todoContent;
+    const waiting = props.todo.waiting;
```

Comment:
```suggestion
function TodoItem ({ todo }) {

    const { deleteTodo, changeTodoStatus } = useTodoList();
    const { id, todoContent, waiting } = todo;
```

`props`가 `todo`외엔 없으니 이렇게 쓰는게 더 좋아보입니다!